src/js/tests/gameModule.test.js

```diff
@@ -0,0 +1,32 @@
+import { gameModule } from '../gameModule';
+import { playerFactory } from '../players';
+
+jest.mock('../players.js');
+
+const mockPlayer0 = { name: 'Bruna', mark: 'X', score: 0 };
+const mockPlayer1 = { name: 'Nicolas', mark: 'O', score: 0 };
+gameModule.setNames(['John', 'Mary']);
+
+
+describe('when new game starts', () => {
+  test("status should be 'on'", () => {
+    playerFactory.mockReturnValueOnce(mockPlayer0);
+    playerFactory.mockReturnValueOnce(mockPlayer1);
+
+    gameModule.newGame();
+
+    expect(gameModule.getStatus()).toBe('on');
+  });
```

Comment:
Like the previous comment, let's cover tests for all the methods of the `gameModule` public methods too (`gameInit`, `newGame`, `makeMove`, `getStatus`, `getCurrentPlayer`, `cleanBoard`, `setNames`, `getPlayer0`, `getPlayer1`).